community/events/index.html

```diff
@@ -20,7 +20,7 @@
         <hr class="whitespace">
         <div class="row">
             <div class="small-12 medium-8 medium-offset-2">
-                <p class="callout primary">[ Hi Helen - please help with this wording. ] We are working to get the rest of the Previous Users Meeting Notes up. In the meantime, previous notes can be found here [insert link].</p>
+                <p class="callout primary">We are in the process of moving content to this new site at present. Details of earlier meetings are still available at [https://www.openmicroscopy.org/site/community/minutes/meetings] </p>
```

Comment:
It's just a placeholder until we decide the strategy - sorry, should have probably said that in the text